ocean_lib/web3_internal/test/test_utils.py

```diff
@@ -33,6 +34,33 @@ def test_prepare_fixed_hash():
     ), "The address is not the expected one."
 
 
+def test_chain_id_send_ether(alice_wallet, bob_address, web3_instance):
+    """Tests if the chainId has the right value for send ether transactions."""
+    receipt = send_ether(alice_wallet, bob_address, 1)
+    if not receipt:
+        raise AssertionError("Send ether was unsuccessful.")
+    tx = web3_instance.eth.get_transaction(receipt["transactionHash"])
```

Comment:
`web3_instance` might have a different chain_id than `alice_wallet.web3`. 

I think you should use `alice_wallet.web3.eth.get_transaction` and `alice_wallet.web3.eth.chain_id`.

Do this for both `test_chain_id_send_ether` and `test_chain_id_cancel_or_replace_transaction`

Reply:
Ok, will try that right now.